### PR TITLE
Update post feature

### PR DESCRIPTION
**Pull request by thaole3107**

### CODE REVIEW

File: src/test/java/com/cg_vibely_social_service/CgVibelySocialServiceApplicationTests.java [hunk 1]
```diff
@@ -4,7 +4,11 @@
 import org.springframework.boot.test.context.SpringBootTest;
 
 @SpringBootTest
+<<<<<<<< HEAD:src/test/java/com/cg_vibely_social_service/VibelySocialApplicationTests.java
```
Comment (chphungphat) [inline src/test/java/com/cg_vibely_social_service/CgVibelySocialServiceApplicationTests.java]: Please resolve conflict code

File: src/test/java/com/cg_vibely_social_service/CgVibelySocialServiceApplicationTests.java [hunk 2]
```diff
@@ -4,7 +4,11 @@
...
+class VibelySocialApplicationTests {
+========
 class CgVibelySocialServiceApplicationTests {
+>>>>>>>> 1261f2bf1f1141a807a673cb59e87e47420e3b50:src/test/java/com/cg_vibely_social_service/CgVibelySocialServiceApplicationTests.java
```
Comment (chphungphat) [inline src/test/java/com/cg_vibely_social_service/CgVibelySocialServiceApplicationTests.java]: Please resolve conflict code

File: src/test/java/com/cg_vibely_social_service/VibelySocialApplicationTests.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
+package com.cg_vibely_social_service;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+<<<<<<<< HEAD:src/test/java/com/cg_vibely_social_service/VibelySocialApplicationTests.java
```
Comment (chphungphat) [inline src/test/java/com/cg_vibely_social_service/VibelySocialApplicationTests.java]: Please resolve conflict code

File: src/test/java/com/cg_vibely_social_service/VibelySocialApplicationTests.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+class VibelySocialApplicationTests {
+========
+class CgVibelySocialServiceApplicationTests {
+>>>>>>>> 1261f2bf1f1141a807a673cb59e87e47420e3b50:src/test/java/com/cg_vibely_social_service/CgVibelySocialServiceApplicationTests.java
```
Comment (chphungphat) [inline src/test/java/com/cg_vibely_social_service/VibelySocialApplicationTests.java]: Please resolve conflict code

File: src/test/java/com/cg_vibely_social_service/CgVibelySocialServiceApplicationTests.java (hunk 1 above)
Comment (thaole3107) [inline src/test/java/com/cg_vibely_social_service/CgVibelySocialServiceApplicationTests.java]: Conflict resolved

File: src/test/java/com/cg_vibely_social_service/VibelySocialApplicationTests.java (hunk 3 above)
Comment (thaole3107) [inline src/test/java/com/cg_vibely_social_service/VibelySocialApplicationTests.java]: Conflict resolved

**Review by chphungphat — APPROVED** [review]

**Review by MinhTam1509 — APPROVED** [review]
i watched and everything is fine

**Review by ChungNguyenVr912 — APPROVED** [review]

**Pull request merged**